### ISSUE TITLE
Multiple background images on first page

### ISSUE THREAD

**Issue by baedorf** [danfickle/openhtmltopdf#649]
I need to combine/place several images on first page using following css:

```css
@page:first {
    @top-center {
         content: 'first';
     }
     background-image: url(background.png), url(image13.png);
}
```

According to css this should be valid and allowed.
Creating a PDF produces following message
````
com.openhtmltopdf.css-parse WARNING:: (f#inline_style_1) Found 2 value(s) for background-image when 1 value(s) were expected at line 52. Skipping declaration.
```` 
How to solve this?

**Comment by danfickle** [comment]
Multiple background images is a CSS3 feature and isn't supported. However I have started implementing it in #650.

**Comment by danfickle** [comment]
Thanks @baedorf. Will be released with 1.0.7. [Background properties on wiki](https://github.com/danfickle/openhtmltopdf/wiki/Big-CSS-reference#background-properties)
